gensim/utils.py

```diff
@@ -1154,29 +1154,29 @@ def keep_vocab_item(word, count, min_count, trim_rule=None):
             return default_res
 
 
-def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
-    """
-    Run command with arguments and return its output as a byte string.
-    Backported from Python 2.7 as it's implemented as pure python on stdlib.
-
-    >>> check_output(args=['/usr/bin/python', '--version'])
-    Python 2.6.2
-    Added extra KeyboardInterrupt handling
+def check_output(args, flag=True):
```

Comment:
A new interface breaks down many wrappers from gensim. First, please merge develop to your branch (it needs to correct test run)

Second, run tests locally with all wrappers like this 
`FT_HOME=/home/ivan/release/test/fastText WR_HOME=/home/ivan/release/test/wordrank  VOWPAL_WABBIT_PATH=/home/ivan/release/test/vowpal_wabbit/vowpalwabbit/vw DTM_PATH=/home/ivan/release/test/dtm/dtm/main MALLET_HOME=/home/ivan/release/test/Mallet python setup.py test`

Another variant is [PR#1368 docker image](https://github.com/RaRe-Technologies/gensim/pull/1368) for this purposes